test/external/test_matrix.py

```diff
@@ -0,0 +1,8 @@
+from pytest import mark
+from playwright.sync_api import APIRequestContext
+
+
+@mark.periodic
+def test_matrix_federation(api_request_context: APIRequestContext):
+    response = api_request_context.get("https://federationtester.matrix.org/api/report?server_name=jupiterbroadcasting.com")
+    assert response.json()["FederationOK"] == True
```

Comment:
I'd suggest putting a few more `assert`s in, so if in the future it fails we have a bit more incite as to where it failed (i.e. if it responded with a 500 instead of a 2xx).

I'd also recommend adding comments after the assert, that way it gives us a summary of what the failure is compared to a snippet of the error message. (I need to correct my test comments, I was commenting what they were not what the error would be :sweat_smile:, but I've included what I'll change mine to in yours )

```suggestion
    assert response.ok, "response code wasn't 200-299"
    assert response.body() != "", "response was empty"
    assert response.json()["FederationOK"] == True
```